content/applications/websites/ecommerce/checkout_payment_shipping/shipping.rst

```diff
@@ -59,30 +59,30 @@ field to add a fixed amount.
    The field used to define additional fees cannot be left empty in your third-party shipping
    provider account.
 
-.. _ecommerce/shipping/custom-method:
+.. _ecommerce/delivery/custom-method:
 
-Custom shipping method
+Custom delivery method
 ======================
 
-Custom shipping methods must be created, for example:
+Custom delivery methods must be created, for example:
 
-- to integrate shipping carriers through :doc:`Sendcloud
+- to integrate delivery carriers through :doc:`Sendcloud
   <../../../inventory_and_mrp/inventory/shipping_receiving/setup_configuration/sendcloud_shipping>`;
 - to configure specific rules (e.g., to offer free shipping for orders above a specific amount) for
   a specific provider;
 - to configure :ref:`Fixed Price <inventory/shipping/fixed>` shipping or shipping
   :ref:`Based on Rules <inventory/shipping/rules>`.
 
-To create a custom shipping method, go to :menuselection:`Website --> Configuration -->
-Shipping Methods`, click :guilabel:`New` and fill in the :ref:`fields
+To create a custom delivery method, go to :menuselection:`Website --> Configuration -->
+delivery Methods`, click :guilabel:`New` and fill in the :ref:`fields
 <inventory/shipping_receiving/shipping-methods-details>`.
 
 In the :guilabel:`Provider` field, select :ref:`Based on Rules <inventory/shipping/rules>`,
 :ref:`Fixed Price <inventory/shipping/fixed>`, or :ref:`Pickup in store <inventory/shipping/pickup>`
 if the shiping method does not involve any specific provider.
 
 .. tip::
-   Upon :ref:`configuring <inventory/shipping_receiving/configure-delivery-method>` a shipping
+   Upon :ref:`configuring <inventory/shipping_receiving/configure-delivery-method>` a delivery
```

Comment:
I can't comment on those lines but lines 90-91 need to be updated (see my comment on line 140 as well):

- use the :guilabel:`Availability` tab to restrict the delivery method's availability;